packages/exceptions/src/http/UriIntegrityException.ts

```diff
@@ -1,6 +1,7 @@
 import { UriException } from './UriException';
 
 /**
+ * @todo need to consider renaming it to DataIntegrityException and use as a common exception
```

Comment:
`DataIntegrityException` is abstract name.
I prefer concrete context-specific exception names, this makes it easy to filter exceptions using `instanceof`

Reply:
removed todo:ok_hand: